Formula/cadence-workflow.rb

```diff
@@ -0,0 +1,29 @@
+class CadenceWorkflow < Formula
+  desc "Cadence Workflow is a distributed, scalable, durable, and highly available orchestration engine to execute asynchronous long-running business logic in a scalable and resilient way"
```

Comment:
```suggestion
  desc "Distributed, scalable, durable, and highly available orchestration engine"
```
This should be enough to fix the audit, I think.

Reply:
Thanks!